backend/src/GraphQl/OpenIdConnect/Authorizations/AuthorizationMutations.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+using System.Security.Claims;
+using System.Threading;
+using System.Threading.Tasks;
+using HotChocolate.Authorization;
+using HotChocolate.Types;
+using Metabase.Authorization;
+using Metabase.Configuration;
+using Metabase.Data;
+using Metabase.Extensions;
+using Metabase.GraphQl.Users;
+using Microsoft.AspNetCore.Identity;
+using OpenIddict.Core;
+using OpenIddict.EntityFrameworkCore.Models;
+
+namespace Metabase.GraphQl.OpenIdConnect.Authorizations;
+
+[ExtendObjectType(nameof(Mutation))]
+public class AuthorizationMutations
+{
+    [UseUserManager]
+    [Authorize(Policy = AuthConfiguration.WritePolicy)]
+    public async Task<DeleteAuthorizationPayload> DeleteAuthorizationAsync(
+        DeleteAuthorizationInput input,
+        ClaimsPrincipal claimsPrincipal,
+        UserManager<User> userManager,
+        OpenIddictAuthorizationManager<OpenIddictEntityFrameworkCoreAuthorization> manager,
+        CancellationToken cancellationToken
+    )
+    {
+        if (input.Id == null)
```

Comment:
`Id` is actually non-nullable. So this should be impossible and thus unnecessary.